spec/system/composer/discard_draft_spec.rb

```diff
@@ -1,68 +1,72 @@
 # frozen_string_literal: true
 
 describe "Composer - discard draft modal", type: :system do
-  fab!(:current_user) { Fabricate(:admin) }
-  fab!(:topic_1) { Fabricate(:topic) }
+  fab!(:topic)
+  fab!(:user) { Fabricate(:admin) }
```

Comment:
This was an issue before, but I much prefer calling this `fab!(:admin)`... it's even ⛳ 

```suggestion
  fab!(:admin)
```

(Would also need changes below.)